portality/static/doaj/js/available_facetviews/admin.background_jobs.facetview.js

```diff
@@ -53,7 +56,7 @@ jQuery(document).ready(function($) {
         if (resultobj.reference) {
             paramsBlock += "<strong>Reference:</strong><br>";
             for (var key in resultobj.reference) {
-                var val = resultobj.reference[key];
+                var val = JSON.stringify(resultobj.reference[key]);
```

Comment:
This puts quote marks around the value when it's a string, which might be confusing, so I may add a catch for arrays and objects and only use JSON.stringify in those cases.